ntfs_fr.md

```diff
@@ -528,6 +529,49 @@ address_id | chaine | Requis | Identifiant de l'adresse
 street_name | chaine | Requis | Nom de la voierie
 house_number | chaine | Optionnel | Numéro du seuil
 
+### occupancies.txt (optionnel)
+La notion d'affluence est appliquée sur l'intervalle entre 2 horaires (affluence à bord du véhicule).
+Pour chaque ligne de ce fichier, l'affluence est appliquée à tous les intervalles d'une circulation entre deux horaires (successifs ou non) remplissant l'ensemble des conditions suivantes :
+
+* le premier des intervalles débute à l'arrêt `from_stop_area` et le dernier des intervalles se termine l'arrêt `to_stop_area` (inclus)
+* le jour de circulation est inclus entre `from_date` et `to_date` (inclus)
+* le jour de semaine de la circulation est actif (voir champs `monday`, `tuesday`, etc.)
+* le `departure_time` du premier intervalle et le `arrival_time` du dernier intervalle sont inclus entre `from_time` et `to_time` (inclus)
```

Comment:
Je déplace la discussion ici pour le confort des réponses.

@woshilapin à écrit:
> @pbench @prhod @pbougue 
> Since `occupancies.txt` has `from_date,to_date` and all the days of the week, would it be wise to reuse the existing `calendar.txt` instead? It doesn't mean we have to use existing calendar, but we could define new ones explicitly for occupancies (and I'm betting sometimes, we're going to reuse existing calendar)? That would simplify a great deal the current file and reuse existing objects.
>
> ```csv
> line_id,from_stop_area,to_stop_area,service_id,from_time,to_time,occupancy
> ```

De base, ça me semble plutôt bien de réutiliser les choses existantes et de mutualiser.
Je vois un souci à ça : le calendar est assez lié (un peu abusivement) aux trips (l'id s'appelle `service_id`).
Je vois un contrepoids (contournement possible ?) à ce souci : le grid_rel_calendar_line qui associe la notion de grid_calendar aux lines, et donc on pourrait envisager le même genre de choses pour l'affluence ?.

Après, je ne suis pas pointu sur la nuance grid-calendar vs. calendar... Mais rendre l'objet calendar multi-purpose me semble être une bonne direction dans la modélisation de plannings :smile_cat: (je ne sais pas s'il y a une meilleure façon de le faire avec des tables de relation-composition, avec rien, avec des "héritages").
Je pense qu'on a un ex-archi-data qui pourra avoir un avis éclairant sur ça !

Reply:
Pour le moment, le modèle sur les taux d'occupation est expérimental, et les calendriers ne seront pas mutualisés. Il y a un risque de "nettoyage" des calendriers considérés comme inutilisés alors qu'ils sont référencés par l'occupation.
Je propose qu'on laisse les deux listes décorellées pour le moment (en indiquant le côté "experimental").

Reply:
Après discussion avec @prhod, c'est pousser l'optimisation un peu loin à ce stade de prototype. On se propose plutôt d'annoter la section entière avec un **Expérimental** histoire que l'API ne soit pas trop figée pour le moment. Il sera toujours temps d'y revenir plus tard.

Reply:
Je résous également cette discussion, car cette PR n'avancera pas plus sur ce sujet comme mentionné précédemment.